lib/http-proxy/passes/web-outgoing.js

```diff
@@ -99,18 +106,21 @@ module.exports = { // <--
 
     // message.rawHeaders is added in: v0.11.6
     // https://nodejs.org/api/http.html#http_message_rawheaders
-    if (proxyRes.rawHeaders != undefined) {
+    if (proxyRes.rawHeaders !== undefined) {
```

Comment:
The current code here supports if it is `undefined` or `null`, we should be sure that it is strictly undefined.